### ISSUE TITLE
clear out work queue when exec command is terminated

### ISSUE THREAD

**Issue by abingham** [sixty-north/cosmic-ray#127]
It seems that when the exec command is terminated the work queue is not cleared out as I (naively) expected. I say this because I see that workers continue processing jobs after I kill the exec. Presumably we can flush the celery queue of the rest of the jobs.

**Comment by abingham** [comment]
From the celery documentation, it looks like we can use `app.control.purge()`.

```
How do I purge all waiting tasks?
Answer: You can use the celery purge command to purge all configured task queues:
$ celery -A proj purge
or programatically:
>>> from proj.celery import app
>>> app.control.purge()
```

**Comment by abingham** [comment]
Fixed here a518723d153826fcd75c27f1bfca3aa67811a139
